SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -136,15 +136,27 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnResponse(Worker_CommandResponseOp&
 	}
 }
 
-void USpatialPlayerSpawner::ObtainPlayerId(FUniqueNetIdRepl& OutUniqueId, FName& OutOnlinePlatformName)
+void USpatialPlayerSpawner::ObtainPlayerId(FURL& LoginURL, FUniqueNetIdRepl& OutUniqueId, FName& OutOnlinePlatformName)
 {
 	const FWorldContext* const WorldContext = GEngine->GetWorldContextFromWorld(NetDriver->GetWorld());
 	check(WorldContext->OwningGameInstance);
 
 	// This code is adapted from PendingNetGame.cpp:242
 	if (ULocalPlayer* LocalPlayer = WorldContext->OwningGameInstance->GetFirstGamePlayer())
 	{
-		// TODO: Send nickname and game login options as a part of the URL. UNR-911
+		// Send the player nickname if available
+		FString OverrideName = LocalPlayer->GetNickname();
+		if (OverrideName.Len() > 0)
+		{
+			LoginURL.AddOption(*FString::Printf(TEXT("Name=%s"), *OverrideName));
+		}
+
+		// Send any game-specific url options for this player
+		FString GameUrlOptions = LocalPlayer->GetGameLoginOptions();
+		if (GameUrlOptions.Len() > 0)
+		{
+			LoginURL.AddOption(*FString::Printf(TEXT("%s"), *GameUrlOptions));
```

Comment:
Can we not just pass the GameUrlOptions directly? Is this just used for consistency with the above format?